client/infinispan-key-value-store-hotrod/src/test/java/org/infinispan/api/client/impl/Person.java

```diff
@@ -0,0 +1,115 @@
+package org.infinispan.api.client.impl;
+
+import java.util.Objects;
+
+import org.infinispan.protostream.annotations.ProtoDoc;
+import org.infinispan.protostream.annotations.ProtoField;
+
+@ProtoDoc("@Indexed")
+public final class Person {
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 1, required = true)
+   String id;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 2, required = true)
+   String firstName;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 3, required = true)
+   String lastName;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 4, required = true)
+   int bornYear;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 5, required = true)
+   String bornIn;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 6, required = true)
+   Address address;
```

Comment:
Same point as Address and required fields. For int you will have to set `defaultValue = x` when the field is not required.

Reply:
I was considering to use the latest proto3 library when it will be released